docs/plugin_development.rst

```diff
@@ -352,48 +427,57 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
    │       └── widgets.py
    └── tests
 
-
 In the root folder you'll find two important files:
 
 * The Makefile, which has several useful commands:
```

Comment:
Definitely going to have to update this old stuff with our standardized Noxfile at some point